packages/layer-selector/LayerSelector.js

```diff
@@ -260,6 +260,13 @@ const LayerSelector = (props) => {
         layerList.remove(managedLayersDraft[layerItem.id].layer);
       }
 
+      // When you set the zoom on a map view without a cached layer, it has no effect on the scale of the map.
+      // This is a hack to re-apply the zoom after adding the first cached layer.
+      if (props.view.scale !== managedLayersDraft[layerItem.id].layer.tileInfo.lods[props.view.zoom].scale) {
+        // eslint-disable-next-line no-self-assign
+        props.view.zoom = props.view.zoom;
```

Comment:
what happens if you add a scale and no zoom property. or no scale and zoom property and only a center point or none of those things?

Reply:
Great point. Addressed in newer commits.

Reply:
What was your 🎳 score?

Reply:
109 on the second game. I'm not a great bowler, but I beat my kids and that's all that mattered. :)

Reply:
since you got over 100, i will merge this.